blog/mindestlohnerhöhung/index.md

```diff
@@ -0,0 +1,52 @@
+---
+title: 12 € Mindestlohn und Minijobgrenze erhöht - was Sie wissen müssen.
+publishDate: 2022-10-15
+slug: Mindestlohnerhöhung
+cover: "münzstapel.png"
+author: jseidel
```

Comment:
Hier fehlt noch die categorie, in diesem fall `aktuelles` (das war auch der grund warum du die fehlermeldung hattest.
```suggestion
author: jseidel
categories:
    - aktuelles
```